### PR TITLE
More info why circle ci could be skipped

### PR DESCRIPTION
**Pull request by janschulz**
As per the discussion in https://github.com/conda-forge/conda-smithy/pull/242/files/8bdc89c7284fa19f6a8d71fc1674123bfff7e0a7#r73888812

### CODE REVIEW

**Comment by jakirkham** [comment]
Thanks @janschulz. LGTM.

File: conda_smithy/templates/circle.yml.tmpl [hunk 1]
```diff
@@ -28,7 +28,7 @@ test:
     # Run, test and (if we have a BINSTAR_TOKEN) upload the distributions.
     - ./ci_support/run_docker_build.sh
 {% else %}
-    # In case the skip fails for some reason, do a fast finish.
+    # The Circle-CI build should not be active but if this is not true for some reason, do a fast finish.
```
Comment (jakirkham) [inline conda_smithy/templates/circle.yml.tmpl]: Very minor grammar note. We should probably have a `,` before `but`. Though it isn't a big deal to me either way.

**Comment by janschulz** [comment]
Will do tomorrow, already in bed right now... :-)

**Comment by janschulz** [comment]
Pushed the typo fix

**Comment by jakirkham** [comment]
Thanks @janschulz.

PS: Normally I'm too timid to merge things here, but I think a comment rephrase is safe. 😄

**Pull request merged**